docs/src/get-started/tutorials/understanding-hypers.py

```diff
@@ -0,0 +1,417 @@
+    f"Descriptor for center_weight={center_weight}: " f"{descriptor.block(0).values[0]}"
+)
+
+# %%
+#
+# Choosing the ``cutoff_function``
+# --------------------------------
+#
+# In a naive descriptor approach all atoms within the cutoff are taken in into
+# account equally and atoms without the cutoff are ignored. This behavior is
+# implemented using the ``cutoff_function={"Step": {}}`` parameter in each
+# calculator. However, doing so means that small movements of an atom near the
+# cutoff result in large changes in the descriptor: there is a discontinuity in
+# the representation as atoms enter or leave the cutoff. A solution is to use
+# some smoothing function to get rid of this discontinuity, such as a shifted
+# cosine function:
+#
+# .. math::
+#
+#     f(r) = \begin{cases}
+#           1 &r < r_c - w,\\
+#           0.5 + 0.5 \cos[\pi (r - r_c + w) / w] &r_c - w < r <= r_c, \\
+#           0 &r_c < r,
+#     \end{cases}
+#
+# where :math:`r_\mathrm{c}` is the cutoff distance and :math:`w` the width.
+# Such smoothing function is used as a multiplicative weight for the
+# contribution to the representation coming from each neighbor one by one
+#
+# The following functions compute such a shifted cosine weighting.
+
+
+def shifted_cosine(r, cutoff, width):
+    """A shifted cosine switching function.
+
+    Parameters
+    ----------
+    r : float
+        distance between neighboring atoms in Å
+    cutoff : float
+        cutoff distance in Å
+    width : float
+        width of the switching in Å
+
+    Returns
+    -------
+    float
+        weighting of the features
+    """
+    if r <= (cutoff - width):
+        return 1.0
+    elif r >= cutoff:
+        return 0.0
+    else:
+        s = np.pi * (r - cutoff + width) / width
+        return 0.5 * (1.0 + np.cos(s))
+
+
+# %%
+#
+# Let us plot the weighting for different widths.
+
+r = np.linspace(1e-3, 4.5, num=100)
+
+plt.plot([0, 4.5, 4.5, 5.0], [1, 1, 0, 0], c="k", label=r"Step function")
+
+for width in [4.5, 2.5, 1.0, 0.5, 0.1]:
+    weighting_values = [shifted_cosine(r=r_i, cutoff=4.5, width=width) for r_i in r]
+    plt.plot(r, weighting_values, label=f"Shifted cosine: $width={width}\\,Å$")
+
+plt.legend()
+plt.xlabel(r"distance $r$ from the central atom in $Å$")
+plt.ylabel("feature weighting")
+plt.show()
+
+# %%
+#
+# From the plot we conclude that a larger ``width`` of the shifted cosine
+# function will decrease the feature values already for smaller distances ``r``
+# from the central atom.
+
+# %%
+#
+# Choosing the ``radial_scaling``
+# -------------------------------
+#
+# As mentioned above all atoms within the cutoff are taken equally for a
+# descriptor. This might limit an ML model's accuracy for atomistic systems on top
+# of a ``cutoff_function``, so it is sometimes useful to weigh neighbors that
+# further away from the central atom less than neighbors closer to the central
+# atom. This can be achieved by a ``radial_scaling`` function with a long-range
```

Comment:
The sentence is a bit unclear, how about

```suggestion
# descriptor. This might limit the accuracy of a model, so it is sometimes useful to weigh neighbors that
# further away from the central atom less than neighbors closer to the central
# atom. This can be achieved by a ``radial_scaling`` function with a long-range
```